### PR TITLE
Update trimming timeline, screenshot tests and documentation

### PR DESCRIPTION
**Pull request by rohangoel96**
Changes proposed in this pull request:
- Update trimTimeline, screenshot tests and documentation
# Checks
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option).
# Any screenshots or GIFs?

If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).
# Notify reviewers

@inodb

### CODE REVIEW

File: js/plugins/trimTimeline.js [hunk 1]
```diff
@@ -133,18 +134,24 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
...
         .attr("stroke", "black")
         .attr("stroke-width", 1)
         .attr("fill", "none")
         .attr("class","kink-line")
-        .attr("transform", "translate("+ (kinkPosition - 92.5) +",20)");
+        .attr("transform", "translate("+ (kinkPosition - 92.5) +",20)")
     }
   }
 
+  d3.select(".kink-bg, .kink-line")
```
Comment (inodb) [inline js/plugins/trimTimeline.js]: This selector would fail if you have two timelines with the kink enabled on the same page

**Comment by inodb** [comment]
@rohangoel96 Thank you!
1. For the trimmed timeline, it seems one can only click on the first zigzag. http://rawgit.com/rohangoel96/clinical-timeline/last-week/index.html?test=data1?view=simple
2. Zoom doesn't seem to be working on the `examples.html` for both FF & Chrome

**Comment by rohangoel96** [comment]
@inodb, ah yes there was a slight fault causing the first one but the examples.html seems to work fine here with me.
ezgif com-crop

**Comment by inodb** [comment]
@rohangoel96 Weird for me it looks like this in both FF & Chrome:

zoom_not_working

In any case, I see in your example as well that it scrolls up a bit if you zoom in. Could you have a look at that?

**Pull request merged**